bin/terminal_velocity

```diff
@@ -96,7 +96,7 @@ the default default will be used"""
     args.exclude = exclude
 
     if args.print_config:
-        print args
+        print(args)
```

Comment:
ok, I guess I also fixed a print for python3, sorry I didn't mention that in the commit (or maybe give it a commit to itself)